src/app/organization/tellers/cashier-template.resolver.ts

```diff
@@ -0,0 +1,31 @@
+/** Angular Imports */
```

Comment:
rename as edit cashier resolver and move to edit cashier folder just like for edit office